### PR TITLE
Add Cradio default

### PR DESCRIPTION
**Pull request by filterpaper**
## Description

Add default layout for Cradio.

### CODE REVIEW

File: public/keymaps/c/cradio_default.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",       "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",
+      "LSFT_T(KC_A)", "LALT_T(KC_S)", "LCTL_T(KC_D)", "LGUI_T(KC_F)", "KC_G", "KC_H", "LGUI_T(KC_J)", "LCTL_T(KC_K)", "LALT_T(KC_L)", "LSFT_T(KC_SCLN)",
+      "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",       "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH",
+                                 "LT(2,KC_TAB)",  "KC_ENT",     "KC_SPC",  "LT(1,KC_BSPC)"
+    ],
+    [
+      "KC_TRNS", "KC_1",    "KC_2",    "KC_3",    "KC_VOLU",    "KC_HOME", "KC_PGDN", "KC_PGUP", "KC_END",  "KC_DQUO",
+      "KC_TRNS", "KC_4",    "KC_5",    "KC_6",    "KC_VOLD",    "KC_LEFT", "KC_DOWN", "KC_UP",   "KC_RGHT", "KC_QUOT",
+      "KC_CAPS", "KC_7",    "KC_8",    "KC_9",    "KC_0",       "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+                                       "MO(3)",   "KC_ESC",     "KC_TRNS", "KC_TRNS"
```
Comment (noroadsleft) [inline public/keymaps/c/cradio_default.json:17]:
```suggestion
                                       "MO(3)",   "KC_GESC",    "KC_TRNS", "KC_TRNS"
```

File: public/keymaps/c/cradio_default.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    [
+      "KC_TRNS", "KC_LBRC", "KC_LCBR", "KC_RCBR", "KC_TRNS",    "KC_CIRC", "KC_LPRN", "KC_RPRN", "KC_RBRC", "KC_TILD",
+      "KC_EXLM", "KC_AT",   "KC_HASH", "KC_DLR",  "KC_PERC",    "KC_AMPR", "KC_MINS", "KC_EQL",  "KC_BSLS", "KC_GRV",
+      "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",    "KC_ASTR", "KC_UNDS", "KC_PLUS", "KC_PIPE", "KC_TRNS",
+                                       "KC_TRNS", "KC_TRNS",    "KC_DEL",  "MO(3)"
+    ],
+    [
+      "KC_TRNS", "KC_F1",   "KC_F2",   "KC_F3",   "KC_F10",     "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "RESET",
+      "KC_TRNS", "KC_F4",   "KC_F5",   "KC_F6",   "KC_F11",     "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_INS",
+      "KC_TRNS", "KC_F7",    "KC_F8",  "KC_F9",   "KC_F12",     "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
```
Comment (noroadsleft) [inline public/keymaps/c/cradio_default.json:28]:
```suggestion
      "KC_TRNS", "KC_F1",   "KC_F2",   "KC_F3",   "KC_F10",     "KC_TRNS", "KC_WH_U", "KC_WH_D", "KC_TRNS", "RESET",
      "KC_TRNS", "KC_F4",   "KC_F5",   "KC_F6",   "KC_F11",     "KC_MS_L", "KC_MS_D", "KC_MS_U", "KC_MS_R", "KC_INS",
      "KC_TRNS", "KC_F7",   "KC_F8",   "KC_F9",   "KC_F12",     "KC_TRNS", "KC_BTN1", "KC_BTN2", "KC_TRNS", "KC_TRNS",
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Updates the keymap so it's the same as running `make cradio:default` from a `qmk_firmware` repository.

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Comment by filterpaper** [comment]
Thank you for the quick review.

**Pull request merged**